RetailSuiteWarehousing/Integration/04_ConfigureStockAllocation.md

```diff
@@ -0,0 +1,101 @@
+[!!Configure the warehouse groups](./01_ConfigureWarehouseGroups.md)
+[!!Configure the warehouses](./02_ConfigureWarehouses.md)
+[!!User interface Warehouse](../UserInterface/03a_Warehouse.md)
+[!!Manage an attribute](../../PIM/Integration/01_ManageAttributes.md)
+[!!Manage an attribute set](../../DataHub/Integration/02_ManageAttributeSets.md)
+
+
+# Configure the stock allocation
+
+The stock allocation function allows you to update the stock levels automatically across all your sales channels. In the stock allocation table, you can define which value should be indicated as available stock in each sales channel. There are different methods you can choose from to calculate the available stock. The calculated stock amount is then transferred to the *PIM* product, where it is displayed in the *Stock level* field. From the *PIM* module, the calculated stock amount is transferred via ETL to the *Omni-Channel* module, and from there to the corresponding sales channel via driver. 
```

Comment:
Erklärung gefällt mir gut
Aber ist die Allocation-Funktion dazu da, den aktuellen Bestand an einen Sales Channel zu schicken? Oder nur, um auf Anfrage in Omni Channel verfügbar zu sein.

Reply:
Wenn alles eingestellt ist, wird auomatisch im Sales channel angezeigt (Lager -> PIM -> OC -> Sales channel)